tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -494,6 +494,51 @@ public function testAlterTableAutoIncrementIntToBigInt(string $from, string $to,
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+
+    public function testGetDetailsWithTwoSameTablesOnDifferentSchemas(): void
+    {
+        $createTableSQL = 'CREATE TABLE migrations(a text);';
+        $this->connection->executeStatement($createTableSQL);
+        $createTableSQL = 'CREATE TABLE "001_test".migrations(a text);';
+        $this->connection->executeStatement($createTableSQL);
+        $this->connection->executeStatement('SET search_path TO "001_test",public');
+
+        $table = new Table('migrations');
+        $table->addColumn('a', 'text');
+
+        try
+        {
+            $databaseTable = $this->schemaManager->listTableDetails('migrations');
+        } catch (\Exception $e) {
+            $databaseTable = null;
+        }
+
+        $c    = new Comparator();
+        $diff = $c->diffTable($table, $databaseTable);
+
+        self::assertNotFalse($diff);
```

Comment:
Is it the only requirement that the diff is not empty?

Reply:
I changed it so diff is not needed anymore.